### ISSUE TITLE
How do you prevent fake voice?

### ISSUE THREAD

**Issue by wannaphong** [TensorSpeech/TensorFlowTTS#372]
Hello. Is it possible to use pretrained from this package to make fake voice by Transfer Learning? How do you prevent fake voice?

**Comment by ZDisket** [comment]
> Is it possible to use pretrained from this package to make fake voice by Transfer Learning?

Depends. How much data do you have, and what is the quality?

>How do you prevent fake voice?

What do you mean?

**Comment by dathudeptrai** [comment]
> Hello. Is it possible to use pretrained from this package to make fake voice by Transfer Learning? How do you prevent fake voice?

you mean the voice generates by voice-cloning technique ?

**Comment by wannaphong** [comment]
> 
> 
> > Hello. Is it possible to use pretrained from this package to make fake voice by Transfer Learning? How do you prevent fake voice?
> 
> you mean the voice generates by voice-cloning technique ?

Yes

**Comment by dathudeptrai** [comment]
you can easily do voice-cloning with this repo if you already have a good speaker encoder (training by deep metric learning, i have a plan to create a speaker identification framework :))). So you want to know how to detect the voice is fake or not ?

**Comment by wannaphong** [comment]
> 
> 
> > Is it possible to use pretrained from this package to make fake voice by Transfer Learning?
> 
> Depends. How much data do you have, and what is the quality?
> 
> > How do you prevent fake voice?
> 
> What do you mean?

If i train pretrained voice with 30hrs dataset,  I am concerned about fake voice that could be caused from pretrained.

**Comment by wannaphong** [comment]
> you can easily do voice-cloning with this repo if you already have a good speaker encoder (training by deep metric learning, i have a plan to create a speaker identification framework :))). So you want to know how to detect the voice is fake or not ?

Yes
